### PR TITLE
Add Support for Fastify v3.x

### PR DESCRIPTION
**Pull request by ms10398**
Submitted to solve #11 

As recently fastify 3.x is released

@StarpTech

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -73,6 +73,6 @@ function fastifyGracefulShutdown(fastify, opts, next) {
 }
 
 module.exports = fp(fastifyGracefulShutdown, {
-  fastify: '^1.0.0 || ^2.0.0',
+  fastify: '^1.0.0 || ^2.0.0 || ^3.0.0',
```
Comment (StarpTech) [inline index.js:76]: It's fine for me to release that change as breaking change. Please only add `^3.0.0`.

Comment (ms10398) [reply index.js:76]: Sure will ditch other changes then

Comment (StarpTech) [reply index.js:76]: Because `fastify-plugin` 2.0.0 only supports fastify 3.0 see https://github.com/fastify/fastify-plugin

Comment (ms10398) [reply index.js:76]: Yes I was upgrading fastify in our project and it broke so headed over here 😅

**Comment by ms10398** [comment]
Have updated the PR @StarpTech

File: index.js (hunk 1 above)
Comment (StarpTech) [inline index.js:76]: I think you misunderstood my review. It's fine to support only `^3.0.0` and add `fastify-plugin@2.0.0` which adds support for fastify v3.

Comment (ms10398) [reply index.js]: Fixed it

**Comment by ms10398** [comment]
Hey @StarpTech so can we go forward and merge it

**Pull request merged**